x-pack/plugins/alerting/server/lib/get_execution_log_aggregation.ts

```diff
@@ -175,7 +175,11 @@ export function getExecutionLogAggregation({
             // Bucket sort to allow paging through executions
             executionUuidSorted: {
               bucket_sort: {
-                sort: formatSortForBucketSort(sort),
+                sort: [{
+                  'numErroredActions>_count': {
+                    order: 'desc',
+                  },
+                }],
```

Comment:
I hard coded the sort here, but we want to enable the sorting via the `ExecutionLogSortFields` type and change the `get_rule_execution_log.ts` route `sortFieldSchema` to include this field